sunpycube/cube/NDCube.py

```diff
@@ -245,18 +248,18 @@ class NDCubeOrdered(NDCube):
     """
     Class representing N dimensional cubes with oriented WCS.
     Extra arguments are passed on to NDData's init.
+    For example, in an x, y, t cube the order would be (t,x,y) and in a
```

Comment:
This should give the complete hierarchy which can be found in cube_utils.orient, I think.  As I remember the order is time, spectral, solar x, solar y, followed by any other dimensions supplied.  But you should double check this.